dockerfiles/alpine/is/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk11:jdk-11.0.10_9-alpine
+FROM adoptopenjdk/openjdk11:jdk-11.0.11_9-alpine
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
       com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.11.0.2"
```

Comment:
Shall we update the latest release tag

Reply:
jdk-11.0.11_9 is the latest release \[1]

\[1] https://hub.docker.com/r/adoptopenjdk/openjdk11/

Reply:
No, This is our Docker IS release tag 
https://github.com/wso2/docker-is/releases/tag/v5.11.0.2
Its mentioned as 5.11.0.2. it should change into v5.11.0.5